columnar/src/column_index/optional_index/set_block/sparse.rs

```diff
@@ -109,3 +109,24 @@ impl<'a> SparseBlock<'a> {
         Err(left)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::column_index::optional_index::set_block::DenseBlockCodec;
+
+    #[test]
+    fn test_simple_translate_idx_to_value_idx_dense() {
+        let mut buffer = Vec::new();
+        DenseBlockCodec::serialize([1, 10].iter().copied(), &mut buffer).unwrap();
```

Comment:
Is this the right file for this test? I would have expected it in dense.rs

Reply:
ah yes, I already moved it, but forgot to delete it